src/deposit/execute_deposit_utils.cairo

```diff
@@ -162,10 +162,9 @@ fn execute_deposit(params: ExecuteDepositParams) {
                 deposit.ui_fee_receiver,
             );
 
-    assert(
-        cache.long_token_amount == 0 && cache.short_token_amount == 0,
-        DepositError::EMPTY_DEPOSIT_AMOUNTS_AFTER_SWAP
-    );
+    if cache.long_token_amount == 0 && cache.short_token_amount == 0 {
```

Comment:
why did you remove the assert?

Reply:
Because it has to be and if not an assert